conf/application-json-logger.xml

```diff
@@ -11,4 +10,4 @@
     <root level="WARN">
         <appender-ref ref="STDOUT"/>
     </root>
-</configuration>
\ No newline at end of file
+</configuration>
```

Comment:
The above `<logger name="application" level="${logger.application:-WARN}"`  can be simplified to ` <logger name="application" level="WARN"/>`